There are 3 errors coming up in the Java Docs build job related to the changes here:

```
[ERROR] /home/jenkins/jenkins-agent/workspace/hadoop-multibranch_PR-3645/ubuntu-focal/src/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/shell/CopyCommandWithMultiThread.java:42: error: malformed HTML
[ERROR]    * set thread count by option value, if the value <= 1, use 1 instead.
[ERROR]                                                     ^
[ERROR] /home/jenkins/jenkins-agent/workspace/hadoop-multibranch_PR-3645/ubuntu-focal/src/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/shell/CopyCommandWithMultiThread.java:53: error: malformed HTML
[ERROR]    * set thread pool queue size by option value, if the value < 1,
[ERROR]                                                               ^
[ERROR] /home/jenkins/jenkins-agent/workspace/hadoop-multibranch_PR-3645/ubuntu-focal/src/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/shell/CommandWithDestination.java:400: error: unknown tag: target
[ERROR]    * file "<target>._COPYING_". If the copy is
```

I guess it does not like the `<` symbol or the `<target>` as it thinks it should be valid HTML.

Could you try to fix those by changing the Java Doc comments and then I think this change is good to commit.

--- a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/shell/CommandWithDestination.java
+++ b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/shell/CommandWithDestination.java
@@ -397,11 +397,11 @@ abstract class CommandWithDestination extends FsCommand {
 
   /**
    * If direct write is disabled ,copies the stream contents to a temporary
-   * file "<target>._COPYING_". If the copy is
-   * successful, the temporary file will be renamed to the real path,
-   * else the temporary file will be deleted.
+   * file "target._COPYING_". If the copy is successful, the temporary file
+   * will be renamed to the real path, else the temporary file will be deleted.
    * if direct write is enabled , then creation temporary file is skipped.
-   * @param in the input stream for the copy
+   *
+   * @param in     the input stream for the copy
    * @param target where to store the contents of the stream
    * @throws IOException if copy fails
    */ 

--- a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/shell/CopyCommandWithMultiThread.java
+++ b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/shell/CopyCommandWithMultiThread.java
@@ -39,7 +39,8 @@ public abstract class CopyCommandWithMultiThread
   public static final int DEFAULT_QUEUE_SIZE = 1024;
 
   /**
-   * set thread count by option value, if the value <= 1, use 1 instead.
+   * set thread count by option value, if the value less than 1,
+   * use 1 instead.
    *
    * @param optValue option value
    */
@@ -50,7 +51,7 @@ public abstract class CopyCommandWithMultiThread
   }
 
   /**
-   * set thread pool queue size by option value, if the value < 1,
+   * set thread pool queue size by option value, if the value less than 1,
    * use DEFAULT_QUEUE_SIZE instead.
    *
    * @param optValue option value

